biz.aQute.bndlib/src/aQute/bnd/osgi/Constants.java

```diff
@@ -115,6 +115,7 @@
 	String				CHECK										= "-check";
 	String				CONDUIT										= "-conduit";
 	String				CONTRACT									= "-contract";
+	String				EMULATE_CONTRACT							= "-emulate-contract";
```

Comment:
Maybe `-define-contract` or `-add-contract` is better than emulate since we are not running or emulating anything.

Reply:
k, I like `-define-contract`